modular_zubbers/maps/offstation/dauntless/mob_spawns.dm

```diff
@@ -12,11 +12,12 @@
 	icon_state = "sleeper_s"
 	prompt_name = "Dauntless Personnel"
 	you_are_text = "You are a syndicate Operative, employed on a top secret spy vessel hidden on lavaland"
-	flavour_text = "Unfortinately, your hated enemy, NanoTrasen, has started mining in this sector. Continue working the best you can, and keep a low profile"
+	flavour_text = "Unfortunately, your hated enemy, NanoTrasen, has started mining in this sector. Continue working the best you can, and keep a low profile" //fixed typo
 	quirks_enabled = TRUE
 	random_appearance = FALSE
 	loadout_enabled = TRUE
 	computer_area = /area/ruin/space/has_grav/bubbers/dauntless/service/dorms
+	banned_species = list(/datum/species/human/felinid/primitive, /datum/species/lizard/ashwalker) //BUBBER EDIT added species restriction for icecat and ashliz
```

Comment:
Probably would be better off making these limits apply to parent and simply override the list with an empty one on the cafe, icemoon hole, and ashie egg spawners

Reply:
Also these can be modular

Reply:
How could these be modular if I'd have to go into each to add the var anyways?

Reply:
Dauntless code is already modular.

Reply:
> How could these be modular if I'd have to go into each to add the var anyways?

Sorry I meant the edits in general I was reviewing this at 1 AM and it shows